.claude/QUICKSTART.md

```
@@ -0,0 +1,170 @@
+- GitHub: https://github.com/settings/tokens
+- Jira: https://id.atlassian.com/manage-profile/security/api-tokens
+
+## 3. Verify
+
+```bash
+cd .claude
+./scripts/setup.sh --verify
+```
+
+Should see:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)